rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -30,12 +30,24 @@ def get_filtered_results(self, start_date, end_date, filters, stratification='EN
     filters_sql_ps = self.get_facets_sql(filters)
     filters_sql_p = self.get_facets_sql(filters, table_prefix='p')
 
+    # check if request stratification is for enromment status
+    query_for_enrollment_status = False
+
+    target_enrollment_statuses = []
+    if 'enrollment_statuses' in filters:
```

Comment:
Is this a new requirement? Why do we need to be able to specify what enrollment statuses to return?

Reply:
It's not a new requirement, just for later use to check the sql execution result.